pallets/loans/src/lib.rs

```diff
@@ -1256,15 +1256,32 @@ impl<T: Config> Pallet<T> {
     ) -> Result<(Liquidity, Shortfall, Liquidity), DispatchError> {
         let total_borrow_value = Self::total_borrowed_value(account)?;
         let total_collateral_value = Self::total_liquidation_threshold_value(account)?;
+
         let base_position = Self::get_base_position(account)?;
+        let dot_borrowed_value = Self::get_asset_value(
+            T::LiquidationFreeAssetId::get(),
+            Self::current_borrow_balance(account, T::LiquidationFreeAssetId::get())?,
+        )?;
```

Comment:
this piece of code duplicates with `get_account_lf_liquidity`

Reply:
dot_* should better be renamed to `lf_*` I guess

Reply:
ah! Sure it is. 😿